cmds/core/sluinit/uinit_linux.go

```diff
@@ -53,6 +100,21 @@ func checkDebugFlag() {
 	}
 }
 
+func parseBootEntryFlag() (string, error) {
+	bootEntry, present := cmdline.Flag(bootEntryFlag)
+	if !present {
+		return "", fmt.Errorf("no boot entry found")
```

Comment:
same thing here. You might want to make a variable
var ErrNoBootEntry = fmt.Errorf("no boot entry found")
Also, on errors, I prefer more info, so
return "", fmt.Errorf("%q:%w", bootEntryFlag, ErrNoBootEntry)
it just make it easier to figure out what's going wrongt.